tests/etl/test_job_loader.py

```diff
@@ -80,20 +81,17 @@ def test_ingest_pulse_jobs(pulse_jobs, test_project, jm, result_set_stored,
     assert JobDetail.objects.count() == 2
 
 
-def test_ingest_pulse_jobs_bad_project(pulse_jobs, test_project, jm, result_set_stored,
-                                       mock_log_parser):
+def test_clean_revision_for_pulse_jobs_bad_project(pulse_jobs):
     """
-    Ingest a job through the JSON Schema validated JobLoader used by Pulse
+    Test cleaning a job with bad repo should throw exception
     """
 
     jl = JobLoader()
-    revision = result_set_stored[0]["revision"]
-    for job in pulse_jobs:
-        job["origin"]["revision"] = revision
-        job["origin"]["project"] = "ferd"
+    job = pulse_jobs[0]
 
-    with pytest.raises(DatasetNotFoundError):
-        jl.process_job_list(pulse_jobs)
+    with JobsModel("ferd") as jm:
+        with pytest.raises(DatasetNotFoundError):
```

Comment:
I may be getting confused here, but isn't the desired behaviour that the test not raise at all? (Perhaps with a check that a log entry was made)

ie: the try-except in job_loader.py needs to be around the whole block.